lua/wikis/commons/OpponentDisplay.lua

```diff
@@ -41,7 +41,7 @@ OpponentDisplay.BracketOpponentEntry = Class.new(
 	function(self, opponent, options)
 		self.content = mw.html.create('div'):addClass('brkts-opponent-entry-left')
 
-		if options.showTbd == false and (Opponent.isEmpty(opponent) or Opponent.isTbd(opponent)) then
+		if options.showTbd == false and (Opponent.isEmpty(opponent) or Opponent.isTbd(opponent, {checkLiterals = true})) then
```

Comment:
```suggestion
		if options.showTbd == false and (Opponent.isEmpty(opponent) or Opponent.isTbd(opponent)) and opponent.type ~= Opponent.literal then
```
wouldn't this do it?

Reply:
well, it has been manually set to TBD then right?

Reply:
hmm
well at least we should check the literal check to tbd and not include empty :P